src/controllers/auth.js

```diff
@@ -219,28 +219,35 @@ const obtainAccessToken = async (req, res) => {
     }
 
     // check if the password is valid
-    let user = users[0];
+    const user = users[0];
     const isMatch = await comparePasswords(password, user.hashed_password);
 
     if (!isMatch) {
       return res.status(401).send({ message: 'Invalid username or password' });
     }
 
-    // get the user roles from the DB
-    const userRoles = rowService.get({
+    // get the users role from the DB
+    const usersRole = rowService.get({
       tableName: '_users_roles',
```

Comment:
Also we used default tables string name in many places, it's better to have them as constants too.

Reply:
All of the table names are not pulled from the `constants` folder